app/client/components/user/CreateUserForm.tsx

```diff
@@ -46,32 +51,39 @@ class CreateUserForm extends React.Component<InjectedFormProps<ICreateUser, Oute
 
     if (!window.IS_USING_BITCOIN) {
       uniqueIdentifierInput = (
-        <Field name="publicSerialNumber" component={TextInput} type="text" label={'Phone Number or ID'}>
+        <InputField name="publicSerialNumber" label={'Phone Number or ID'} isRequired>
           <QrReadingModal
             updateData={ (data: string) =>  this.setSerialNumber(data) }
           />
-        </Field>
+        </InputField>
       )
     } else {
-      uniqueIdentifierInput = <Field name="blockchainAddress" component={TextInput} type="text" label='Bitcoin Address' />
+      uniqueIdentifierInput = <InputField name="blockchainAddress" label='Bitcoin Address' isRequired />
     }
 
-
+    //TODO(admin_create): set a default for it to be easier for staff? should it be 400 or 40000 if we're giving 400 CICs?
```

Comment:
@enjeyw if we want to give 400 tokens, is it initial disbursement of 400 or 40000? and should we just set it default to be that instead of having them type it every time?

Reply:
so on the _user interface_ it's 400, but it's stored everywhere as 400,00 (aka the amounts in cents). There's actually already a default setting, thats triggered through "make_initial_disbursement" when the user is approved. It's currently set through the config file via "STARTING_BALANCE", though it should probably be migrated to a setting associated with an organisation.

Reply:
eh so STARTING_BALANCE is added automatically? do we still need the initial disbursement amount input field then? and agreed, starting balance should be moved to the organisation level instead of being a global

Reply:
huh... i've looked through the user creation code now. so there's both starting balance AND custom initial disbursement. are they used differently? should we just remove the custom initial disbursement from the admin user creation form?

Reply:
custom initial disbursement is useful if the admin wants to set up a user with an amount that isn't the default, which is pretty often, though the UI should probably do a better job of suggesting it's not default

Reply:
@enjeyw got it, going to change it to "additional_initial_disbursement" in that case